code/game/objects/effects/decals/cleanable/food.dm

```diff
@@ -31,15 +31,12 @@
 	icon_state = "salt_pile"
 
 /obj/effect/decal/cleanable/food/salt/Crossed(mob/living/L)
-	if(is_species(L, /datum/species/snail))
+	if(is_species(L, /datum/species/snail) || is_species(L, /datum/species/squid))
 		while(L.loc == src.loc)
 			L.adjustFireLoss(2, TRUE)
+			if(is_species(L, /datum/species/squid)
+				L.visible_message("<span class='danger'>Your tentacles shrivel as you step over the salt! It burns!</span>")
```

Comment:
Remove the second check for squid species and just make it a more generic message, something like "The salt! It burns!". Also this should be a `to_chat()`. 

You could also maybe add a `prob()` check to scream.

Reply:
This prob() check to scream is nice, will find it in code shortly